src/fastman/utils.py

```
@@ -0,0 +1,180 @@
+        if not init_file.exists():
+            init_file.touch()
+
+        return path
+
+    @staticmethod
+    def write_file(path: Path, content: str, overwrite: bool = False) -> bool:
+        """Write file with safety checks"""
+        try:
+            with path.open('x' if not overwrite else 'w', encoding='utf-8') as f:
+                f.write(content)
+            return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)